### PR TITLE
Closes #193 | Create dataset loader for MALINDO Morph

### LINKED ISSUE #193: Create dataset loader for MALINDO_Morph
Dataloader name: `malindo_morph/malindo_morph.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?malindo_morph

|     Dataset| malindo_morph |
|-------------|---|
| Description | MALINDO Morph merupakan kamus morfologi untuk bahasa Melayu dan bahasa Indonesia. Kamus MALINDO Morph dilesenkan dengan pelesenan Creative Commons Attribution 4.0 International (CC BY 4.0). Untuk maklumat terperinci mengenai MALINDO Morph, sila rujuk makalah di bawah ini. |
| Subsets | Kamus Dewan, Kamus Besar Bahasa Indonesia, Leipzig Corpora Collection, Frogstory-David, Melayu-Standard-Lisan, Melayu-Sabah, Melayu-Sarawak, Melayu-Brunei, Indo-Jakarta-Lisan |
| Languages | zlm, ind |
| Tasks | Morphological Inflections |
| License | Creative Commons Attribution 4.0 (cc-by-4.0) |
| Homepage | https://github.com/matbahasa/MALINDO_Morph |
| HF URL | - |
| Paper URL | http://lrec-conf.org/workshops/lrec2018/W29/pdf/8_W29.pdf |

### PR DESCRIPTION
**Pull request by danjohnvelasco**
Closes #193

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [ ] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

> Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.

This dataset is a lexicon. It does not support any tasks. This case is similar to https://github.com/SEACrowd/seacrowd-datahub/pull/155

To run tests:
```
python -m tests.test_seacrowd_source_only seacrowd/sea_datasets/malindo_morph/malindo_morph.py
```

### CODE REVIEW

File: seacrowd/sea_datasets/malindo_morph/malindo_morph.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  isbn = {979-10-95546-24-5},
+  language = {english}
+  }
+"""
+
+
+_DATASETNAME = "malindo_morph"
+
+_DESCRIPTION = """\
+MALINDO Morph is a morphological dictionary for Malay (bahasa Melayu) and Indosian (bahasa Indonesia) language.
```
Comment (holylovenia) [inline seacrowd/sea_datasets/malindo_morph/malindo_morph.py:51]: "Indosian" --> "Indonesian"

**Review by holylovenia — APPROVED** [review]
Other than one minor fix below, I have nothing more to suggest. LGTM!

**Comment by danjohnvelasco** [comment]
> Other than one minor fix below, I have nothing more to suggest. LGTM!

Fixed!

**Comment by holylovenia** [comment]
Replacing @gentaiscool with @jen-santoso due to no response.

**Review by jen-santoso — CHANGES_REQUESTED** [review]
Hi @danjohnvelasco, thank you for the dataloader implementation.

So this case is a source-only, right? The dataloader runs well, just two nits here and the dataloader's good to go!

**Comment by danjohnvelasco** [comment]
> Hi @danjohnvelasco, thank you for the dataloader implementation.
> 
> So this case is a source-only, right? The dataloader runs well, just two nits here and the dataloader's good to go!

Hi @jen-santoso, thanks for the review. Yes, this is source-only. Your suggested changes are commited.

File: seacrowd/sea_datasets/malindo_morph/malindo_morph.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+MALINDO Morph is a morphological dictionary for Malay (bahasa Melayu) and Indonesian (bahasa Indonesia) language.
+It contains over 200,000 lines, with each containing an analysis for one (case-sensitive) token.
+Each line is made up of the following six items, separated by tabs: root, surface form, prefix, suffix, circumfix, reduplication.
+"""
+
```
Comment (jen-santoso) [inline seacrowd/sea_datasets/malindo_morph/malindo_morph.py:21]: The description is duplicate from `_DESCRIPTION`, so let's remove this part
```suggestion
```

**Review by jen-santoso — CHANGES_REQUESTED** [review]
Sorry, one last nit. Other than that, it's LGTM

**Review by jen-santoso — APPROVED** [review]

**Pull request merged**